core/validation.go

```diff
@@ -173,16 +167,6 @@ func (app *App) validateOrders(orders []*zeroex.SignedOrder) (*zeroex.Validation
 			})
 			continue
 		}
-		maxExpiration := big.NewInt(time.Now().Add(maxOrderExpirationDuration).Unix())
-		if order.ExpirationTimeSeconds.Cmp(maxExpiration) > 0 {
-			results.Rejected = append(results.Rejected, &zeroex.RejectedOrderInfo{
-				OrderHash:   orderHash,
-				SignedOrder: order,
-				Kind:        zeroex.MeshValidation,
-				Status:      zeroex.ROMaxExpirationExceeded,
```

Comment:
Let's remove the definition for `zeroex.ROMaxExpirationExceeded` since I think it's no longer used.